x-pack/plugins/profiling/server/routes/setup.ts

```diff
@@ -51,13 +52,12 @@ export function registerSetupRoute({
       try {
         const esClient = await getClient(context);
         const core = await context.core;
-        const clientWithDefaultAuth = createProfilingEsClient({
+        const client = createProfilingEsClient({
           esClient,
           request,
-          useDefaultAuth: true,
         });
         const setupOptions: ProfilingSetupOptions = {
-          client: clientWithDefaultAuth,
+          client,
```

Comment:
Shouldn't we have both clients in this case?

I mean, for the setup options other than the data check, I think it does make sense to use our local stack. As we want to install the packages in our local environment. But for the data check it does make sense to use the cluster, if provided, to check if there is any profiling data.

I imagine that to test the setup functionality locally we'd get this response from the `resources` API:
```
has_data: true, //Uses cluster information to check if there's any profiling data
has_setup: false, //Uses local stack to check if profiling was already setup
```
Once we click on the setup button, this would be the response:
```
has_data: true, //Uses cluster information to check if there's any profiling data
has_setup: true, //Our local stack has been set up to use profiling (we can still reset everything to test it again)
```

Reply:
@jbcrail any update on this?

Reply:
No updates at this time. Due to the 8.9 feature freeze and data model versioning for 8.10, this issue dropped in priority on my list at the moment.